event-processing/event-processor/tests/unit/app.test.ts

```diff
@@ -45,10 +45,6 @@ describe('Unit test for app handler', function () {
 
         const sqsEvent = TestHelper.createSQSEventWithEncodedMessage(TestHelper.encodeAuditEvent(exampleMessage));
 
-        await expect(handler(sqsEvent))
```

Comment:
Are we not throwing an error here anymore?

Reply:
No, because I've moved the timestamp formatting out of the validation, the validation logic simply returns false if the timestamp is invalid.

Reply:
ah okay makes sense